traffic_monitor/cache/cache.go

```diff
@@ -300,6 +301,14 @@ func (handler Handler) Handle(id string, rdr io.Reader, format string, reqTime t
 	}
 
 	stats, miscStats, err := decoder.Parse(result.ID, rdr, pollCtx)
+	if val, ok := miscStats["current_time_epoch_ms"]; ok {
+		valString := fmt.Sprintf("%s", val)
+		valInt, err := strconv.ParseInt(valString, 10, 64)
+		if err != nil {
```

Comment:
If this conversion fails, should it still try to convert `valInt` into a time value on line 310? Shouldn't it just error out like in line 314?